docs/60-gpu.md

```diff
@@ -0,0 +1,8 @@
+# GPU
+
+Use GPUs to accelerate your Spark applications. By default, when you specify an Azure VM type that has a GPU, your Docker image will contain CUDA-8.0 and cuDNN-6.0. See [Docker Image](./12-docker-image.md) for more information about the AZTK Docker images.
+
+AZTK uses Nvidia-Docker to expose the VM's GPU(s) inside the container. Nvidia drivers (ver 384) are installed at runtime.
+
+### Installation Location
+By default, CUDA is installed at `/usr/local/cuda-8.0`.
```

Comment:
Can you add a sample or tutorial section?


# Tutorial

Create a cluster specifying a GPU enabled VM
```sh
aztk spark cluster create --id gpu-cluster --vm-size standard_nc6 --size 1
```

Submit your an application to the cluster that will take advantage of the GPU
```sh
aztk spark cluster submit --id gpu-cluster --name gpu-app ./examples/src/main/python/gpu/nubma_example.py
```